CHANGELOG.md

```diff
@@ -34,7 +36,8 @@ The file format is based on [Keep a Change Log](https://keepachangelog.com/en/1.
 - loading a game that doesn't exist no longer crashes the game
 - rangefinder fixed
 - fixed the '$' sign in the shop
-- Fix smoke particles not being removed after starting a new game
+- fix smoke particles not being removed after starting a new game
```

Comment:
Since Smoke.destroy is commented out the particles won't get removed when starting a new game.